docs/user_manual/projects.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: page
+title: "6. Projects"
+section: 6
+---
+
+
+A project contains information about a set of studies that may comprise many different samples, experiments and
```

Comment:
Since studies aren't mandatory in MISO, maybe this could be reworded to lessen the emphasis on studies? Moderate preference.